packages/client/src/components/BottomNavBar/Right-bottom-navigation/BottomNavigation.component.js

```diff
@@ -11,21 +11,26 @@ const BottomNavigation = () => {
     setSearchInputValue(e.target.value);
   }
 
-  const handleSubmit = (e) => {
+  const keyPressHandler = (e) => {
```

Comment:
I don't think you need this anymore, because the Enter click event bubbles up to the form handler.